src/SonarQube.Client/Api/V7_20/GetIssuesRequestWrapper.cs

```diff
@@ -61,6 +64,8 @@ public async Task<SonarQubeIssue[]> InvokeAsync(HttpClient httpClient, Cancellat
             innerRequest.Branch = Branch;
             innerRequest.Logger = Logger;
             innerRequest.IssueKeys = IssueKeys;
+            innerRequest.RuleId = RuleId;
+            innerRequest.ComponentKey = ComponentKey;
 
             if (innerRequest.IssueKeys != null)
             {
```

Comment:
I wonder why we do it in 3 different server requests below and not just one with multiple issue types. Also, I'm not sure the new method needs to return taints.

Reply:
it is to circumvent the limit on the Server.